qa/compare_snapshots.py

```diff
@@ -0,0 +1,38 @@
+import click
+import requests
+
+from qa.comparitor import Comparitor
+
+@click.command("compare-snapshots")
```

Comment:
Can this live in `cli/`? maybe we need more documentation around it - and pyseir is currently the exception - but all of the other cli interfaces live in `cli/`